04-onion-routing.md

```diff
@@ -270,6 +276,7 @@ The writer of the TLV `payload`:
     - MUST NOT include any other tlv field.
   - For every node outside of a blinded route:
     - MUST include `amt_to_forward` and `outgoing_cltv_value`.
+    - SHOULD include `attributable_error` if the full path supports it.
```

Comment:
Make the legacy case more restrictive?
```
if every node in the route advertises `option_attributable_error`: 
  - SHOULD include `attributable_error` for every hop
otherwise: 
  - MUST NOT include `attributable_error`
 ```

Reply:
Yes, good to make that explicit. Updated.